README.md

```diff
@@ -1,10 +1,10 @@
 # CoBlocks: Page Builder Blocks for Gutenberg
 
-[![CircleCI](https://circleci.com/gh/godaddy/coblocks.svg?style=svg)](https://circleci.com/gh/godaddy/coblocks) [![WordPress plugin](https://img.shields.io/wordpress/plugin/dt/coblocks.svg?style=flat)](https://wordpress.org/plugins/coblocks/) [![WordPress plugin](https://img.shields.io/wordpress/plugin/v/coblocks.svg?style=flat)](https://wordpress.org/plugins/coblocks/) [![WordPress](https://img.shields.io/wordpress/v/coblocks.svg?style=flat)]() [![License](https://img.shields.io/badge/license-GPL--2.0%2B-red.svg)](https://github.com/richtabor/coblocks/blob/master/license.txt)
+[![CircleCI](https://circleci.com/gh/godaddy/coblocks.svg?style=svg)](https://circleci.com/gh/godaddy-wordpress/coblocks) [![WordPress plugin](https://img.shields.io/wordpress/plugin/dt/coblocks.svg?style=flat)](https://wordpress.org/plugins/coblocks/) [![WordPress plugin](https://img.shields.io/wordpress/plugin/v/coblocks.svg?style=flat)](https://wordpress.org/plugins/coblocks/) [![WordPress](https://img.shields.io/wordpress/v/coblocks.svg?style=flat)]() [![License](https://img.shields.io/badge/license-GPL--2.0%2B-red.svg)](https://github.com/godaddy-wordpress/coblocks/blob/master/LICENSE)
```

Comment:
[![CircleCI](https://circleci.com/gh/godaddy/coblocks.svg?style=svg)]
Should be
[![CircleCI](https://circleci.com/gh/godaddy-wordpress/coblocks.svg?style=svg)]

```
[![CircleCI](https://circleci.com/gh/godaddy/coblocks.svg?style=svg)]
Should be
[![CircleCI](https://circleci.com/gh/godaddy-wordpress/coblocks.svg?style=svg)]
```

Reply:
@richtabor In my PR https://github.com/godaddy-wordpress/coblocks/pull/1162 (which I've closed) I used the following, which sets the default branch to master:
`[![CircleCI](https://circleci.com/gh/godaddy-wordpress/coblocks/tree/master.svg?style=svg)](https://circleci.com/gh/godaddy-wordpress/coblocks/tree/master)`

Reply:
Thanks!